Classes/GarbageCollector.php

```diff
@@ -155,6 +155,20 @@ public function processDatamap_afterDatabaseOperations($status, $table, $uid, ar
 			|| ($table == 'pages' && !$this->isIndexablePageType($record))
 		) {
 			$this->collectGarbage($table, $uid);
+
+			// if the start time is in the future and not otherwise invisible,
+			// it is necessary to add the item to the queue again to be indexed once the start time is reached
+			if ($this->isStartTimeInFuture($table, $record)
+				&& !$this->isHidden($table, $record)
+				&& !$this->hasFrontendGroupsRemoved($table, $record)
+				&& ($table != 'tt_content' && $table != 'pages_language_overlay')
```

Comment:
Because if it is tt_content or pages_language_overlay there is an updateItem done instead of deleteItem anyway.

However, I think we could just execute the updateItem again?

Paranheses is not necessary but personally I wanted to add the "grouping" because for those two tables the updateItem is exeucted anyway instead of deleteItem.